pkg/common/cluster_actions.go

```diff
@@ -59,7 +61,13 @@ func (i *ClusterActionRunner) Create(obj runtime.Object) error {
 		return err
 	}
 
-	return i.client.Create(i.context, obj)
+	err = i.client.Create(i.context, obj)
+	if err != nil {
+		return err
+	}
+
+	i.cr.UpdateStatusSecondaryResources(i.cr, reflect.TypeOf(obj).String(), obj.(v1.Object).GetName())
```

Comment:
Does anyone have any suggestions on a better wat to get the kind other than `reflect.TypeOf(obj).String()`? This currently outputs "*v1.Service" for example with the `*`.

Reply:
Using `reflect` is the only way I know...

Reply:
not sure what the code is doing , you could also use ``` search.GetObjectKind().GroupVersionKind().Kind``` I believe

Reply:
Thanks @maleck13 . These methods do exist, unfortunately the Kind is an empty string. I believe because this is initial creation and is using our Golang objects with the type set but no specific Kind field.